spid-spring-integration/src/main/java/it/italia/developers/spid/integration/util/SPIDIntegrationUtil.java

```diff
@@ -130,6 +135,13 @@ public String printAuthnRequest(AuthnRequest authnRequest) throws IntegrationSer
 			log.error("printAuthnRequest :: " + e.getMessage(), e);
 			throw new IntegrationServiceException(e);
 		}
+		
+		try {
+			Signer.signObject(authnRequest.getSignature());
+		} catch (SignatureException e) {
+			  e.printStackTrace();
```

Comment:
Consider using an error log instead of printStackTrace(), for example:
```suggestion
			  log.error("There was an error while signing the request", e);
```

Reply:
Added logging statement as well as throwing an exception